.github/workflows/build.yml

```diff
@@ -91,18 +95,21 @@ jobs:
           path: ..
 
       - name: tests
+        # not sure why `success() ||` is needed here and in step `lint`...
```

Comment:
The Actions docs write 

> If your if expression does not contain any of the status functions it will automatically result with success()

But we have other stages that don't have status functions, and don't automatically return success(). So I'm not sure. What behavior were you seeing without `success() ||`?

The logic looks right to me (even if `success()` should be unnecessary), and the tests work, so this is probably good.

Reply:
Ah, I should have mentioned this. Without `success() ||`, the lint step was getting skipped if the test step failed. [example test](https://github.com/bryangingechen/mathlib/runs/538762874?check_suite_focus=true), [commit](https://github.com/bryangingechen/mathlib/commit/9bd8b4fe5b6dd88445cd03366a7d227598bd5eff).

Reply:
Oh, of course, that makes sense. Without the status function, the step is automatically skipped if any previous step failed. We could equivalently write `always() &&`.